src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -89,24 +92,33 @@ export class SkyFileAttachmentComponent implements AfterViewInit, AfterContentIn
     return this._disabled;
   }
 
-/**
- * Specifies the maximum size in bytes for valid files.
- */
+  /**
+   * Specifies the maximum size in bytes for valid files.
+   */
   @Input()
   public maxFileSize: number = 500000;
 
-/**
- * Specifies the minimum size in bytes for valid files.
- */
+  /**
+   * Specifies the minimum size in bytes for valid files.
+   */
   @Input()
   public minFileSize: number = 0;
 
+  /**
+   * Specifies a custom validation function: `[validateFn]="validateFile"`.
```

Comment:
What is this validating? Should we specify why users would want to provide a custom validation function?

Reply:
I don't think we should try to specify why a user would use this. They basically would use this any time they need to check for something to do with the file that we don't check for (such as the file name being in a given format). The cases for this are probably too large to try to spell out and custom validation is something that is fairly standard with forms. This component is just kind of strange in how you have to do it. Normally this custom validation would be baked into the form but because of how this component works we have to use this input instead. 

I've updated the description here slightly. Let me know if you want to chat more about it.

Reply:
I defer to you on how much to say. I was just checking to see if we needed to provide more here. I made a couple suggestions to the wording, but feel free to disregard the suggestions if they don't work.